Makefile

```diff
@@ -929,7 +929,7 @@ dist/test/c/Hacl_Test_K256.c: KRML_EXTRA=-drop Lib.IntTypes.Intrinsics -add-incl
 
 copy-krmllib:
 	mkdir -p dist/karamel
-	(cd $(KRML_HOME) && tar cvf - krmllib/dist/minimal $$(find include -not -name 'steel_types.h')) | (cd dist/karamel && tar xf -)
```

Comment:
@R1kM the `find` invocation was also finding directories, so even though individual files were not included, the `include/krml` directory still appears, which tar interpreted as meaning "all the files in that directory")